src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/NotificationsServiceTests.cs

```diff
@@ -108,23 +134,31 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
                 var notificationsService = BuildNotificationsService(
                     contentDbContext, storageQueueService.Object);
 
-                await notificationsService.NotifySubscribersIfApplicable(release1.Id, release2.Id, release3.Id);
+                await notificationsService.NotifySubscribersIfApplicable(release1.Id, release2.Id, amendedRelease1.Id);
 
                 storageQueueService.Verify(mock => mock.AddMessages(
                     PublicationQueue,
                 new List<PublicationNotificationMessage>
                     {
                         new()
                         {
-                            Name = release1.Publication.Title,
                             PublicationId = release1.Publication.Id,
-                            Slug = release1.Publication.Slug,
+                            PublicationName = release1.Publication.Title,
+                            PublicationSlug = release1.Publication.Slug,
+                            ReleaseName = release1.Title,
+                            ReleaseSlug = release1.Slug,
+                            Amendment = false,
+                            UpdateNote = "No update note provided.",
```

Comment:
`UpdateNote` values of `No update note provided.` vs. `latest update note` won't be tested here as it's not defined as a member in equals/hashcode of `PublicationNotificationMessage`.

Reply:
I've added `UpdateNote` in `GetHashCode()`